esmvaltool/install/Julia/setup.jl

```diff
@@ -5,7 +5,7 @@ if VERSION >= v"0.7.0-DEV.2005"
     using Pkg
 end
 
-ENV["PYTHON"] = string(ENV["CONDA_PREFIX"], "/bin/python")
+# ENV["PYTHON"] = string(ENV["CONDA_PREFIX"], "/bin/python")
```

Comment:
Is this not needed?

Reply:
It was failing for me and the installation worked without it

Reply:
Let's just remove the line then
```suggestion
```